yaml/overlays/dev/kustomize.sh

```diff
@@ -194,6 +194,7 @@ calico_create_template ()
   export CALICOVPP_VPP_STARTUP_SLEEP=${CALICOVPP_VPP_STARTUP_SLEEP:=0}
   export CALICOVPP_TAP_RX_QUEUES=${CALICOVPP_TAP_RX_QUEUES:=1}
   export CALICOVPP_TAP_TX_QUEUES=${CALICOVPP_TAP_TX_QUEUES:=1}
+  export CALICOVPP_TAP_MTU=${CALICOVPP_TAP_MTU:=0}
```

Comment:
Same here we should default to 1440

Reply:
This might be debatable, `0` will call VPP's api with `mtu_set = false` so afaik use linux's default (which in the end might be 1500). Depends on what's best for most usecases.